extensions/aerial-photo/tests/aerial_photo_collection.test.js

```diff
@@ -29,4 +29,39 @@ o.spec('Aerial Photo Extension Collection', () => {
     // then
     o(valid).equals(true)(JSON.stringify(validate.errors, null, 2));
   });
+  o("Summaries with no 'aerial-photo:run' property should fail validation", async () => {
+    // given
+    const example = JSON.parse(await fs.readFile(examplePath));
+    delete example['summaries']['aerial-photo:run'];
```

Comment:
generally square bracket notation is only used for things that cant be just accessed `['summaries'] -> example.summaries`